android/libraries/rib-base/src/main/java/com/badoo/ribs/core/view/ViewFactoryOld.kt

```diff
@@ -0,0 +1,4 @@
+package com.badoo.ribs.core.view
+
+@Deprecated("Use new template from RIBs 0.3+ that has proper DI for views")
+interface ViewFactoryOld<View>: ViewFactory<Nothing?, View>
```

Comment:
should we add `ReplaceWith("ViewFactory<>")` to deprecation annotations?

Reply:
no, because ViewFactory should no longer be used directly in place where the old one was, but rather in ViewImpl.Factory, hence just the referral to new templates